packages/@tinacms/graphql/src/resolve.ts

```diff
@@ -38,23 +26,8 @@ import { optimizeDocuments } from '@graphql-tools/relay-operation-optimizer'
 
 import type { GraphQLResolveInfo } from 'graphql'
 import type { Database } from './database'
-import type { TinaCloudCollection } from './types'
-import { Path } from 'graphql/jsutils/Path'
 import { NAMER } from './ast-builder'
 
-type VisitorType = Visitor<ASTKindToNode, ASTNode>
```

Comment:
Pretty much all of the deletions here are because the new `useGraphQLForms` doesn't need this info any more. We were previously keeping track of what parts of the graph were different nodes and we no longer need to do that.